src/app/search/state/search.service.spec.ts

```diff
@@ -138,17 +138,13 @@ describe('SearchService', () => {
 
     const b: Workflow = {
       ...a,
-      author: 'B',
+      authors: null,
       full_workflow_path: 'Bcd',
       starredUsers: [{ isAdmin: false, curator: false, platformPartner: null, setupComplete: true }],
     };
 
-    const c: Workflow = { ...a, author: null, full_workflow_path: null, descriptorType: Workflow.DescriptorTypeEnum.WDL };
+    const c: Workflow = { ...a, authors: null, full_workflow_path: null, descriptorType: Workflow.DescriptorTypeEnum.WDL };
```

Comment:
These changes were to fix the unit test issues due to the deprecation of Author in another PR.

Reply:
Final removal to be clear. 
author was already deprecated in favour of authors